utilities/rocAL/rocAL_unittests/pixel_comparison/image_comparison.py

```diff
@@ -70,71 +60,71 @@ def main():
     golden_file_path = ""
     for aug_name in rocal_output_dir_list:
         temp = aug_name.split('.')
-        file_name_s = temp[0].split('_')
-        if(len(file_name_s) > 3):
-            file_name_s.pop()
-            golden_file_path = "_".join(file_name_s) + ".png"
+        file_name_split = temp[0].split('_')
+        if(len(file_name_split) > 3):
+            file_name_split.pop()
+            golden_file_path = "_".join(file_name_split) + ".png"
         else:
             golden_file_path = aug_name
 
         #For randomized augmentation
-        if(file_name_s[0] in randomized_augmentation):
-            total_case_count=total_case_count+1
+        if(file_name_split[0] in randomized_augmentation):
+            total_case_count = total_case_count+1
             augmentation_name = aug_name.split('.')[0]
             logging.info("Running %s",augmentation_name)
-            passed_case_count=passed_case_count+1
+            passed_case_count=passed_case_count + 1
             logging.info("PASSED ")
         elif golden_file_path in golden_output_dir_list:
-            total_case_count=total_case_count+1
-            ref_file_path = ref_output_path+golden_file_path
-            rocal_file_path = rocal_output_path+aug_name
+            total_case_count = total_case_count+1
+            ref_file_path = ref_output_path + golden_file_path
+            rocal_file_path = rocal_output_path + aug_name
             if(os.path.exists(rocal_file_path) and os.path.exists(ref_file_path)):
                 logging.info("Running %s ",aug_name.split('.')[0])
                 img1 = Image.open(ref_file_path)
                 img2 = Image.open(rocal_file_path)
 
                 # Check if the images have the same dimensions
                 if img1.size != img2.size:
-                    logging.info("The images have different sizes.")
+                    logging.info("Golden output and augmentation outputs are having different sizes. Exiting!")
```

Comment:
Change this to 
Mismatch in sizes between the golden outputs and the augmentation outputs.